lms/templates/courseware/accordion.html

```diff
@@ -36,7 +36,7 @@ <h3 ${active_class} aria-label="${aria_label}">
               <p class="subtitle">${section['format']} ${due_date}</p>
 
               % if 'graded' in section and section['graded']:
-                  <img src="/static/images/graded.png" alt="Graded Section">
+              <i class="icon fa fa-pencil-square-o" aria-hidden="true" data-tooltip="${_("This section is graded.")}"></i>
```

Comment:
Let's use `<span>` instead of `<i>`.

FYI: `<i>` has for some poor reason become the standard for icon font use. This element's semantic meaning is to italicize or emphasize, and using the wrong element trips up automated tests.